callcommands/sv_callcommands.lua

```diff
@@ -142,11 +142,20 @@
             RegisterServerEvent('SonoranCAD::callcommands:SendCallApi')
             AddEventHandler('SonoranCAD::callcommands:SendCallApi',
                 function(emergency, caller, location, description, source, silenceAlert, useCallLocation, t)
+					local vec3Call = false
+					local locationVec3 = nil
+					if type(location) == 'vector3' then 
+						vec3Call = true 
+						locationVec3 = location
+					end
+					
                     local postal = ""
                     if location == '' then
                         location = LocationCache[source] ~= nil and LocationCache[source].location or 'Unknown'
-                    elseif type(location) == 'vector3' then
-                        postal = getPostalFromVector3(location)
+                    elseif vec3Call then
+						local jsonPostal = exports['nearest-postal']:getPostalServer(location)
```

Comment:
Some users may not be using nearest-postal, please use the PostalCache and leave in the check for the postals plugin.

Reply:
Perhaps there's a different bug related to the function that you all were using but i couldn't get it to work.

Reply:
@nathanctech commit made with requested changes